draft-packages/brand/docs/Brand.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react"
+import { Brand } from "../KaizenDraft/Brand/Brand"
+
+export default {
+  title: "Brand (React)",
+  component: Brand,
+  parameters: {
+    docs: {
+      description: {
+        component: 'Import { Brand } from "@kaizen/draft-brand"',
+      },
+    },
+  },
+}
+
+const reversedLogoStyles = {
+  width: "200px",
+  backgroundColor: "#e6e5e5",
+  padding: "20px",
+}
+
+export const DefaultSiteDemo = () => (
+  <div style={{ width: "200px" }}>
```

Comment:
Thinking out loud here - do we definitely want to leave the sizing completely to the consumer, or is this something that should be a prop sent into the component 🤔

Reply:
Is this not just for the stories? Or does this get used in production too?